docker-compose.yml

```diff
@@ -151,6 +151,8 @@ services:
       context: ./
       dockerfile: ./build/docker/native.dockerfile
     image: datadog-native
+    environment: 
+    - buildConfiguration=${buildConfiguration}
```

Comment:
Should it default here too? It doesn't remove the need to default on the shell file since the script should stand on its own but gets "more" consistency.

Reply:
Defaults are in `.env` file